src/components/About/styles.scss

```diff
@@ -0,0 +1,125 @@
+@import "SHARED/fonts.scss";
+
+
+@media only screen and (min-width: 320px) {
+	.wrapper {
+		width: 100%;
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+		background-color: blue;
+		--portrait-spacing: 10vw;
+
+		.image {
+			width: 100%;
+			height: 38%;
+		}
+
+		.textContainer {
+			width: 100%;
+			height: 62%;
+			padding-top: var(--portrait-spacing);
+			padding-right: var(--portrait-spacing);
+			padding-bottom: 0;
+			padding-left: var(--portrait-spacing);
+			overflow-y: auto;
+			background-color: pink;
+
+			.heading {
+				margin-bottom: 1rem;
+				font-size: 1.15rem;
+			}
+
+			.subheading {
+				margin-bottom: 1.5rem;
+				font-size: calc(1.15rem * 1.62);
```

Comment:
This is a magic number - we should indicate where it's coming from by using variables:

```
font-size: calc(var(--font_size__heading) * 1.62)
```